doc/developers/contributing.rst

```diff
@@ -548,8 +548,8 @@ message, the following actions are taken.
     [cd build gh]          CD is run only for GitHub Actions
     [cd build cirrus]      CD is run only for Cirrus CI
     [lint skip]            Azure pipeline skips linting
-    [scipy-dev]            Build & test with our dependencies (numpy, scipy, etc ...) development builds
-    [nogil]                Build & test with the nogil experimental branches of CPython, Cython, NumPy, SciPy...
+    [scipy-dev]            Build & test with our dependencies (numpy, scipy, etc) development builds
```

Comment:
Shouldn't it be
```suggestion
    [scipy-dev]            Build & test with our dependencies (numpy, scipy, etc.) development builds
```
to be consistent with your suggestions below?

Reply:
I think, you're right, I changed it :-)